### PR TITLE
[inductor] Revert channels-last support

### PR DESCRIPTION
**Pull request by anijain2305**
We witnessed slow compilation times last week. Earlier, I thought it was due to parallel compilation. But, after git bisect, I found the source of extra time to be my PR - https://github.com/pytorch/pytorch/pull/87049

For 1x1 kernel, the current striding check incorrectly declares channels-first 1x1 convs to channels last. I am not sure why it caused so much compilation time jump.  Or why it did not fail? There was no change in performance speedup. cc @jansel @lezcano @fdrocha @mlazos @soumith @voznesenskym @yanboliang @penguinwu to identify what could be source of this compilation time increase, so that we can manually check that part of the stack.

With this `res2next50` compilation time went back to 96 seconds (which was raised to 900 seconds with my earlier PR) for single thread. And parallel-compilation brings it down to ~30 seconds.

### CODE REVIEW

**Review by soumith — APPROVED** [review]

**Review by jansel — APPROVED** [review]

**Comment by ngimel** [comment]
We'll need to follow up on this, @anijain2305 what's the repro command for the res2next50?

**Comment by anijain2305** [comment]
This PR still errors out for 1x1 convs. @Chillee has volunteered to help out with this.

**Comment by ngimel** [comment]
Update on this: increase in compile time is due to a lot of pointwise kernels that need to be autotuned (now they are tiled hence require autotuning, previously they weren't), and Animesh's https://github.com/pytorch/pytorch/pull/87588/commits/017e3d3bf047ee4f25639e6d394c4a43cc40cd6c fixes that. However, even with reverting original channels last change the tests are still failing, in a weird way, where ref - ref_fp64 is very large (that shouldn't depend on whatever we are doing with inductor). 
So, bottom line: https://github.com/pytorch/pytorch/pull/87588/commits/017e3d3bf047ee4f25639e6d394c4a43cc40cd6c or disabling channels-last as now both fail tests, for unknown reasons, https://github.com/pytorch/pytorch/pull/87588/commits/017e3d3bf047ee4f25639e6d394c4a43cc40cd6c is better because it still leaves channels-last enabled.

**Review by ngimel — APPROVED** [review]

**Comment by desertfire** [comment]
Confirmed this will bring the TIMM model running time back to previously seen before the migration. Let's prioritize to land this, and bring back TIMM model test afterwards. Other proper fixes can go after that.
